src/enums/roleTypes.ts

```diff
@@ -4,6 +4,7 @@ export enum RoleTypes {
   CUSTODIAN = 'Custodian',
   INCORPORATOR = 'Incorporator',
   LIQUIDATOR = 'Liquidator',
+  PARTNER = 'Partner',
```

Comment:
Did the API update? I'm still seeing lowercase at the moment, but i think it's moving towards this? Can't recall what we settled on.

Reply:
@vysakh-menon-aot Will this work or does the API need a change?

You can see here that, currently, a registration saves role types in Title Case:

![image](https://user-images.githubusercontent.com/10002889/162832347-29ff27e2-f089-4630-b5b6-70f9ba9179a2.png)

Reply:
This will work. And in case of edit-ui the parties endpoint change should be in dev now (as lekshmi worked on that ticket today)